src/Core/Models/SystemRegisterRequest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Altinn.Platform.Authentication.Core.Models;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+namespace Altinn.Platform.Authentication.Core.SystemRegister.Models
+{
+    /// <summary>
+    /// Model for the System Register
+    /// A Registered System is a product supplied by a System Vendor,
+    /// it may need Rights to use or acccess Resources at a Service Provider.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public class SystemRegisterRequest
+    {
+        /// <summary>
+        /// The unique Id for this product, in human-readable string format.
+        /// The id is in the format of system_vendor_name_plus_name_chosen_by_them.
```

Comment:
The id summary comment should be changed to prefix of Orgnr, not name. As we discussed last week.